src/main/resources/net.fabricmc.fabric.mixins.common.json

```diff
@@ -9,6 +9,8 @@
     "container.MixinServerPlayerEntity",
     "entity.MixinEntityTracker",
     "entity.MixinEntityTrackerEntry",
+    "entity.MixinLivingEntity",
+    "entity.damage.MixinDamageTracker",
```

Comment:
I believe all the current ones are in alphabetical order, might be nice to also do this here?

Reply:
`Apple` comes after `A` in the dictionary/alphabetical order, thus `entity.damage` comes after `entity`.

Reply:
Additionally, the `damage` package is within the `entity` package, so it makes more sense to come after.

Reply:
ok, this is fine. It was a super minor thing as well.